app/presenters/idv/how_to_verify_presenter.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+class Idv::HowToVerifyPresenter
+  include ActionView::Helpers::TagHelper
+  include ActionView::Helpers::TranslationHelper
+
+  attr_reader :idv_session
```

Comment:
Is this being used anywhere?

Reply:
Resolved with commit 11ab0eb94b37b93e6dac68fb5898b63b74ba795c